tldr.sh

```diff
@@ -72,57 +72,60 @@ init_term_cmds() {
             white=$(    tput setaf 7 || tput AF 7    )
 
             onblue=$(   tput setab 4 || tput AB 4    )
-            ongrey=$(   tput setab 7 || tput AB 7    )
-        }
+            ongrey=$(   tput setab 7 || tput AB 7    );;
+        esac
     } 2>/dev/null ||:
 
     # osx's termcap doesn't have italics. The below adds support for iTerm2
     # and is harmless on Terminal.app
     [ "$(get_platform)" = "osx" ] && {
-        italic=$(echo -e "\033[3m")
-        eitalic=$(echo -e "\033[23m")
+        italic=$(printf "\033[3m")
+        eitalic=$(printf "\033[23m")
     }
 }
 
 heading() {
     local line="$*"
-    echo "$bold$red${line:2}$reset"
```

Comment:
So, when I originally wrote the coloring and formatting I was matching the official tldr output. Maybe I got it wrong? Same comment with the other color and style changes below.